js/functions.js

```diff
@@ -0,0 +1,19 @@
+// Функция для проверки длины строки
+
+const checkRandomString = (string, length) => string.length <= maxLength;
```

Comment:
const checkRandomString = (string, maxLength) => string.length <= maxLength
```suggestion
const checkRandomString = (string, length) => string.length <= maxLength;
```